<ui version="4.0" >
 <class>ForwardConnectInfoDialog</class>
 <widget class="QDialog" name="ForwardConnectInfoDialog" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>275</width>
    <height>231</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Port Forward Info Dialog</string>
  </property>
  <layout class="QVBoxLayout" >
   <item>
    <widget class="QGroupBox" name="groupBox" >
     <property name="title" >
      <string>Information:</string>
     </property>
     <layout class="QGridLayout" >
      <item row="0" column="0" >
       <widget class="QLabel" name="label" >
        <property name="text" >
         <string>Forward Host:</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1" >
       <widget class="QLineEdit" name="lineEdit" >
        <property name="text" >
         <string>218.244.130.188</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0" >
       <widget class="QLabel" name="label_2" >
        <property name="text" >
         <string>User name:</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1" >
       <widget class="QLineEdit" name="lineEdit_2" >
        <property name="text" >
         <string>webroot</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0" >
       <widget class="QLabel" name="label_3" >
        <property name="text" >
         <string>Passowrd:</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1" >
       <widget class="QLineEdit" name="lineEdit_3" />
      </item>
      <item row="3" column="0" >
       <widget class="QLabel" name="label_4" >
        <property name="text" >
         <string>Listen Port:</string>
        </property>
       </widget>
      </item>
      <item row="3" column="1" >
       <widget class="QLineEdit" name="lineEdit_4" >
        <property name="text" >
         <string>6000</string>
        </property>
       </widget>
      </item>
      <item row="4" column="0" >
       <widget class="QLabel" name="label_5" >
        <property name="text" >
         <string>Forward To Port:</string>
        </property>
       </widget>
      </item>
      <item row="4" column="1" >
       <widget class="QLineEdit" name="lineEdit_5" >
        <property name="text" >
         <string>22</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" >
     <item>
      <widget class="QPushButton" name="pushButton_3" >
       <property name="text" >
        <string>&amp;Help</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer>
       <property name="orientation" >
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" >
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton" >
       <property name="text" >
        <string>&amp;Ok</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_2" >
       <property name="text" >
        <string>&amp;Cancel</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>pushButton</tabstop>
  <tabstop>lineEdit_2</tabstop>
  <tabstop>lineEdit_3</tabstop>
  <tabstop>lineEdit_4</tabstop>
  <tabstop>lineEdit_5</tabstop>
  <tabstop>pushButton_3</tabstop>
  <tabstop>lineEdit</tabstop>
  <tabstop>pushButton_2</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>pushButton</sender>
   <signal>clicked()</signal>
   <receiver>ForwardConnectInfoDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>147</x>
     <y>211</y>
    </hint>
    <hint type="destinationlabel" >
     <x>98</x>
     <y>184</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_2</sender>
   <signal>clicked()</signal>
   <receiver>ForwardConnectInfoDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>247</x>
     <y>200</y>
    </hint>
    <hint type="destinationlabel" >
     <x>274</x>
     <y>197</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
